standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/utils/HiveStrictManagedUtils.java

```diff
@@ -59,6 +59,10 @@ public static String validateStrictManagedTable(Configuration conf,
         if (isAvroTableWithExternalSchema(table)) {
           return createValidationError(table, "Managed Avro table has externally defined schema.");
         }
+      } else if (tableType == TableType.EXTERNAL_TABLE) {
+        if (MetaStoreServerUtils.isTransactionalTable(table.getParameters())) {
+          return createValidationError(table, "Table is marked as a external table but it is transactional.");
+        }
```

Comment:
Do we have a testcase for a external table created with transaction = true?

Reply:
duplicate comment.